### ISSUE TITLE
upper case keywords, false positive

### ISSUE THREAD

**Issue by larshp** [larshp/abaplint#422]
```abap
SELECT foo
  UP TO @bar ROWS
  INTO CORRESPONDING FIELDS OF TABLE @boo
  FROM loo.
```
